scripts/getReleaseStatus.test.js

```diff
@@ -91,8 +101,8 @@ describe('Get release status script', () => {
   })
 
   it('should store the data it gets if others fail', async () => {
-    const [residentialLocationUrl, ...restUrls] = allUrls
```

Comment:
the first is not residential location and eslint complains about variable name shadowing